scripts/webpack/package-loader-rule.js

```diff
@@ -1,22 +1,30 @@
 const path = require('path');
+
 const { projects } = require('../../workspace.json');
 
 const tsSrcPackages = [];
+const esbuildSrcPackages = [];
 const aliasMap = {};
 
-for (const [packageName, packagePath] of Object.entries(projects)) {
-    const packageSrcPath =  path.join(__dirname, '../../', `${packagePath}/src`);
+Object.entries(projects).forEach(([packageName, packagePath]) => {
+    const packageSrcPath = path.join(__dirname, '../../', `${packagePath}/src`);
 
-    tsSrcPackages.push({
+    // New esbuild-loader rules for all packages
+    esbuildSrcPackages.push({
         test: /\.[tj]s$/,
         include: packageSrcPath,
-        loader: 'ts-loader',
+        loader: 'esbuild-loader',
+        options: {
+            target: 'es2018', // Modern target without IE 11
```

Comment:
Let's align the version with `checkout-js` first as our first step, which is `es2015`